src/ray/raylet/worker.h

```diff
@@ -71,11 +73,35 @@ class Worker {
   void DirectActorCallArgWaitComplete(int64_t tag);
   void WorkerLeaseGranted(const std::string &address, int port);
 
-  /// Cpus borrowed by the worker. This happens when the machine is oversubscribed
-  /// and the worker does not get back the cpu resources when unblocked.
-  /// TODO (ion): Add methods to access this variable.
-  /// TODO (ion): Investigate a more intuitive alternative to track these Cpus.
-  ResourceSet borrowed_cpu_resources_;
+  // Setter, geter, and clear methods  for allocated_instances_.
+  void SetAllocatedInstances(TaskResourceInstances &allocated_instances) { 
+      allocated_instances_ = allocated_instances;
+  };                                                   
+  TaskResourceInstances &GetAllocatedInstances() {return allocated_instances_; };    
```

Comment:
Add newlines between method definitions please. Same with others in this file.

Reply:
done